collect_app/src/main/java/org/odk/collect/android/utilities/ApplicationConstants.java

```diff
@@ -32,7 +32,7 @@
             "ca", "cs", "de", "en", "es", "et", "fa", "fi", "fr", "ha",
             "hi", "hu", "in", "it", "ja", "ka", "km", "lo_LA", "lt", "mg", "mr",
             "my", "nb", "ne_NP", "nl", "no", "pl", "ps", "pt", "ro", "ru", "so",
-            "sq", "sw", "sw_KE", "ta", "th_TH", "tl", "tr", "uk", "ur",
+            "sq", "sw", "sw_KE", "ta", "th_TH", "ti", "tl", "tr", "uk", "ur",
```

Comment:
Thanks for remembering to do this. I also added it in Transifex at https://www.transifex.com/opendatakit/collect/language/ti/